### ISSUE TITLE
Discovery Filter - 'Exclude Genre' option and Filter Profiles

### ISSUE THREAD

**Issue by Jarsky** [sct/overseerr#3293]
### Description

**Exclude Genre**

It would be great if we could do an "exclude" in the filter.
I've tried using the filter, but its absolutely full of anime/animation and kids shows, even though I havent selected those category.  Unless I explicitely choose Networks I know dont do kids shows/anime. 
I assume because they exist in both anime/kids as well as general genre

**Filter Profiles**

It would be great if we could also save Discovery filter "profiles" either at a Global Level by Administrator, or at User Level where Users can define them themselves. 

### Desired Behavior

Open the Discovery Filter, select categories I want to view, and also select categories I exclusively dont want to view and see more relevant results. 

The ability to select from several predefined Discovery filters from a dropdown, so we don't have to define the filters every time. 

### Additional Context

Applying these filter, and using an exclude for in my case, animation and kids, it should get rid of all of these from Discovery suggestions. 

![image](https://user-images.githubusercontent.com/839416/215300232-0c31185e-e2cf-40ef-9e9e-e2608d7a1d88.png)

### Code of Conduct

- [X] I agree to follow Overseerr's Code of Conduct

**Comment by sct** [comment]
This is two separate requests. There won't be "predefined" filters (in the way you are describing), but we will add exclusion options.

**Comment by derekpurdy** [comment]
I agree with the exclude filter, I am the opposite of OP. I have no need to see the 93% of my discover and movie suggestions are horror movies. I don't download that genre because my kids use my plex, but when I log into overseer it's always just horror movie reccomendations because I have none. To be able to exlcude a genre would be great.

**Comment by oOStroudyOo** [comment]
I also would like this.
I want to exclude Horrors and Kids films.

**Comment by Jarsky** [comment]
Hi team, wondering if theres any progress on adding an Exclusion filter? 🙂

**Comment by rmatec** [comment]
Half my feed is now anime. Any update on this please?

**Comment by DrBazUK** [comment]
Would be great to be able to remove large sections of recommendations that I have zero interest in.

**Comment by sylekta** [comment]
I would very much like to exclude Talk shows, any progress on this?
